templates/projects/view.tmpl

```diff
@@ -1,10 +1,10 @@
 {{$canWriteProject := and .CanWriteProjects (or (not .Repository) (not .Repository.IsArchived))}}
 
 <div class="ui container tw-max-w-full">
-	<div class="tw-flex tw-justify-between tw-items-center tw-mb-4 tw-gap-3">
-		<h2 class="tw-mb-0 tw-flex-1 tw-break-anywhere">{{.Project.Title}}</h2>
-			<div class="project-toolbar-right">
-				<div class="ui secondary filter menu labels">
+	<div class="flex-text-block tw-flex-wrap tw-mb-4">
+		<h2 class="tw-mb-0">{{.Project.Title}}</h2>
+		<div class="tw-flex-1"></div>
+				<div class="ui secondary menu tw-m-0">
```

Comment:
This block looks incorrectly intended, it should be on the same level as the empty div above.

Reply:
No, i mean this:

```hbs
		<div class="tw-flex-1"></div>
				<div class="ui secondary menu tw-m-0">
```

Line 2 should dedent, old code did not have the closing `</div>` tag that you introduced.

Reply:
This line  `		<div class="tw-flex-1"></div>`  is newly added, it doesn't affect the tags .  The newly added line has correct indent

Reply:
I don't care enough about intendation on a soon to be obsolete branch and main branch looks correct.

Reply:
Yes, main branch is correct because I did extra formatting.

This PR is only a backport and has to be done manually, I just don't want to introduce too many other changes